docker/kind/kind-config.yaml

```diff
@@ -4,4 +4,5 @@ featureGates:
   "CRIContainerLogRotation": true
 nodes:
   - role: control-plane
-  - role: worker
\ No newline at end of file
+  - role: worker
```

Comment:
This configures kind to use 2 workers so that we can observe the agents handing over event handling duties when one is destroyed